### DISCUSSION TITLE
BUG Report : Updates overview missing

### DISCUSSION THREAD

**Discussion by Roseghost** [alkem-io/alkemio#520] [General]
![image](https://user-images.githubusercontent.com/64191883/152313844-b09834c4-4545-4ef4-93e0-04e000e1f942.png)

See All option is missing to get to this page https://xd.adobe.com/view/38a5f5f3-8bbb-4f91-9f32-d5edd4fd9d55-6df6/screen/69284d04-afc5-4a12-af80-cf0a064ce159 

Just like discussion 
![image](https://user-images.githubusercontent.com/64191883/152314043-efc4dbc1-7a4e-4c8f-930b-0e04d1094cb5.png)

**Comment by Comoque1** [comment]
Hi Rose, thanks for your feedback/
This is going to be available with the upcoming release.

**Comment by Comoque1** [comment]
Just FYI, fix is deployed.
![image](https://user-images.githubusercontent.com/23620658/152348141-062429ed-ee7b-44ae-ac6e-69433205c4dc.png)
